src/InputWithLabel.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect, useRef } from "react";
+
+const InputWithLabel = ({ todoValue, onInputChange, children }) => {
+  const inputRef = useRef();
+  useEffect(() => {
+    inputRef.current.focus();
+  }, [todoValue]);
+
+  return (
+    <>
+      <label htmlFor="todoTitle">{children}</label>
+      <input
+        name="title"
+        id="todoTitle"
+        value={todoValue}
+        onChange={onInputChange}
+        ref={inputRef}
+      ></input>
```

Comment:
Nitpick:
All input elements can be a self closing tag since it'll never contain children: 
```<input ... />```